### PR TITLE
chore: address pr comments from pr 1008

### PR DESCRIPTION
**Pull request by kevinslin**
https://github.com/dendronhq/dendron/pull/1008

### CODE REVIEW

File: packages/plugin-core/src/_extension.ts [hunk 1]
```diff
@@ -157,6 +157,16 @@ async function startServerProcess(): Promise<{
...
+  if (getStage() !== "prod") {
+    const out = await launchv2({
+      logPath: path.join(__dirname, "..", "..", "dendron.server.log"),
+    });
+    return {port: out.port}
+  }
+
+  // start server is separate process
   const logPath = DendronWorkspace.instance().context.logPath;
   const out = await ServerUtils.execServerNode({
```
Comment (jonathanyeung) [inline packages/plugin-core/src/_extension.ts:171]: do we need to similarly put this inside an `else{` block; i.e. only launch the background process if we're prod?

Comment (kevinslin) [reply packages/plugin-core/src/_extension.ts:171]: nope. we should never hit this in development

**Review by jonathanyeung — APPROVED** [review]
Approval for posterity sake; changes have already been merged though.

File: packages/plugin-core/src/_extension.ts (hunk 1 above)
Comment (jonathanyeung) [reply packages/plugin-core/src/_extension.ts:171]: Ah, misread the code logic.

**Pull request closed without merging**